bootstrap/bootstrap_base.py

```diff
@@ -0,0 +1,30 @@
+# Databricks notebook source
+# MAGIC %run ./install_benvy
+
+# COMMAND ----------
+
+from benvy.databricks.repos import bootstrap  # noqa
+from benvy.databricks.detector import is_databricks_repo  # noqa
+
+if is_databricks_repo():
+    bootstrap.install()
+
+# COMMAND ----------
+
+from benvy.databricks.repos import bootstrap  # noqa
+from benvy.databricks.detector import is_databricks_repo  # noqa
+
+if is_databricks_repo():
+    bootstrap.setup_env()
+
+# COMMAND ----------
+
+import os
+
+if "APP_ENV" not in os.environ:
+    os.environ["APP_ENV"] = "dev"
```

Comment:
This is becomi ng pretty dangerous as it is not hidden even "deeper", don't you think?

Reply:
Let's leave it here, but emit some kind of logging.warning("Using default DEV environment. Please set APP_ENV on cluster and remove the lines above")

Reply:
Don't forget to put the warning into skeleton-databricks as well